spec/cucumber/cli/profile_loader_spec.rb

```diff
@@ -42,6 +42,25 @@ def loader
         given_cucumber_yml_defined_as yml
         expect(loader.args_from('default')).to eq ['--format','pretty','features/sync_imap_mailbox.feature:16:22']
       end
+
+      it 'correctly parses a profile that uses tag expressions (with double quotes)' do
+        given_cucumber_yml_defined_as('default' => '--format "pretty" features\sync_imap_mailbox.feature:16:22 --tags "not @jruby"')
+        if(Cucumber::WINDOWS)
+          expect(loader.args_from('default')).to eq ['--format','pretty','features\sync_imap_mailbox.feature:16:22','--tags','not @jruby']
+        else
+          expect(loader.args_from('default')).to eq ['--format','pretty','featuressync_imap_mailbox.feature:16:22','--tags','not @jruby']
+        end
+      end
+
+      it 'correctly parses a profile that uses tag expressions (with single quotes)' do
+        given_cucumber_yml_defined_as('default' => "--format 'pretty' features\\sync_imap_mailbox.feature:16:22 --tags 'not @jruby'")
```

Comment:
@mattwynne From the irb:
```
irb(main):004:0> puts 'dir\file'
dir\file
=> nil
irb(main):005:0> puts "dir\file"
dir
   ile
=> nil
irb(main):006:0> puts "dir\\file"
dir\file
=> nil
```
Within `""` the `\` is interpreted as "escape" so if you literally mean `\` you have to use `\\`, which when the `""` expression is "evaluted" is "converted" to `\`. (`\f` in the `""` expression above means ["formfead"](https://stackoverflow.com/a/31791519))